doc/tutorials/content/gpu_octree.rst

```diff
@@ -15,7 +15,7 @@ The primary aim of the GPU octree module is to allow parallelized search functio
 This module primarily provides implementations for the following three search methods.
 
 - Radius search
-- K Nearest Neighbors Search (currently restricted to k=1 only)
+- K Nearest Neighbors Search
 - Approximate Nearest Neighbors Search
```

Comment:
You should use singular form. Only the first one is returned.

Reply:
Okay. But K nearest neighbour sounds a bit wrong :sweat:

Reply:
Wrong line. I'm talking about the Approximate Nearest Neighbor item.

Reply:
Ahh. Thats allright then. :sweat_smile: